platform/search/bundle.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Search",
+    "description": "Allows the user to search through the file tree.",
+    "extensions": {
+        "constants": [
+            {
+                "key": "GENERIC_SEARCH_ROOTS",
```

Comment:
The basic rule of thumb I'm trying to cover is "reasonable default behavior, with a clear path for overriding default behavior if necessary".  I think this goes with the hollywood principle, too.

Additional roots (in some deployments, "Packets") wouldn't be included by default, even though the default functionality we want for "Packets" is generic search.  This is why I describe this as "opt-in".